### PR TITLE
Test of pull request

### PR DESCRIPTION
**Pull request by cpannet**
Eh there

### CODE REVIEW

**Comment by cpannet** [comment]
If there is anyone here, hello !

File: cyrilpannetier-planetX.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+A planet X is hidding in the Solar System. Maybe it has plenty of free crêpes, cider and beers.
+
+Apparently, we'll discover it very soon.
```
Comment (PabloRdrRbl) [inline cyrilpannetier-planetX.txt:3]: For example, I could ask you for a modification in this line before accepting your review. But I won't be mean :)

Good luck with your planet hunt Cyril!

**Review by ivalaginja — APPROVED** [review]
Ready to merge.

**Review by PabloRdrRbl — APPROVED** [review]
Now, I approve.

**Pull request merged**